webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/RoleEntity.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.usermanagement;
+
+import io.camunda.webapps.schema.entities.AbstractExporterEntity;
+import java.util.Set;
+
+public class RoleEntity extends AbstractExporterEntity<RoleEntity> {
+
+  private Long roleKey;
+  private String name;
+  private Set<Long> assignedMemberKey;
+
+  public Long getRoleKey() {
+    return roleKey;
+  }
+
+  public RoleEntity setRoleKey(final Long roleKey) {
+    this.roleKey = roleKey;
+    return this;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public RoleEntity setName(final String name) {
+    this.name = name;
+    return this;
+  }
+
+  public Set<Long> getAssignMemberKey() {
```

Comment:
```suggestion
  public Set<Long> getAssignMemberKeys() {
```

Reply:
@Ben-Sheppard - I remember chatting with you and talking about how these fields & methods should be singular and not plural. Should I be using `getAssignedMemberKeys` or `getAssignMemberKey` here?

Reply:
As its a list we should go with `getAssignedMemberKeys` (plural) I may have mispoken is we spoke about it before, apologies